modules/twinklexfd.js

```diff
@@ -266,6 +266,8 @@ Twinkle.xfd.callback.wrongVenueWarning = function twinklexfdWrongVenueWarning(ve
 		case 'rm':
 			if (namespace === 14) { // category
 				text = 'Please use CfD or CfDS for category renames.';
+			} else if ([118, 119, 2, 3].indexOf(namespace) > -1) { // draft, draft talk, user, user talk
+				text = 'RMs are not permitted in draft and userspace, unless they are technical requests.';
```

Comment:
```suggestion
				text = 'RMs are not permitted in draft and userspace, unless they are uncontroversial technical requests.';
```

To make the wording in line with the "uncontroversial technical request" checkbox